src/components/apiTest/apiTest.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { getUserById, getUsers } from './apiHelper';
+
+export default function ApiTest() {
+  const [users, setUsers] = useState([]);
+  const [error, setError] = useState('');
+  const [loding, setLoding] = useState(false);
+
+  const fetchUsers = async() => {
+    try {
+      setError('');
+      setLoding(true);
+      setUsers(await getUsers());
```

Comment:
약간 취향 차이일 수 있는데, 아래 처럼 쓰면 조금 더 직관적이라고 생각합니다.
```suggestion
      const response = await getUsers();
      setUsers(response);
```

Reply:
오 훨씬 직관적이네요! 감사합니다!